src/lib/dev-env/dev-env-core.js

```diff
@@ -296,44 +205,6 @@ async function prepareLandoEnv( instanceData, instancePath ) {
 	debug( `Lando file created in ${ landoFileTargetPath }` );
 }
 
-async function landoRunWp( instancePath, args ) {
-	const lando = new Lando( getLandoConfig() );
-	await lando.bootstrap();
-
-	const app = lando.getApp( instancePath );
-	await app.init();
-
-	const isUp = await isEnvUp( app );
-
-	if ( ! isUp ) {
-		throw new Error( 'environment needs to be started before running wp command' );
-	}
-
-	const wpTooling = app.config.tooling?.wp;
-	if ( ! wpTooling ) {
-		throw new Error( 'wp is not a known lando task' );
-	}
-
-	/*
-	 lando is looking in both passed args and process.argv so we need to do a bit of hack to fake process.argv
-	 so that lando doesn't try to interpret args not meant for wp.
-
-	 Lando drops first 3 args (<node> <lando> <command>) from process.argv and process rest, so we will fake 3 args + the real args
-	*/
-	process.argv = [ '0', '1', '3' ].concat( args );
-
-	wpTooling.app = app;
-	wpTooling.name = 'wp';
-
-	const wpTask = landoBuildTask( wpTooling, lando );
-
-	const argv = {
-		_: args,
-	};
-
-	wpTask.run( argv );
-}
-
 export function generateInstanceData( slug: string, options: NewInstanceOptions ) {
 	const instanceData = {
```

Comment:
I know this was not introduced in the PR, but we could return the object directly and not declare the variable.

Reply:
That function will be totally rewritten in the prompt work I am doing now.